`tldr sort` tries to find the `tldr-sort` command

This was extremely weird. I didn't bother to even try and find the source of this strange bug but I might later on.

`tldr sort` threw me the following error after correctly displaying the page for sort:
```
module.js:340
    throw err;
          ^
Error: Cannot find module '/usr/local/bin/tldr-sort'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

This is just a bit silly, right? I tried `tldr ls` and it didn't have the problem.

--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
   "dependencies": {
     "async": "~0.9.0",
     "colors": "~1.0.1",
-    "commander": "~2.3.0",
+    "commander": "~2.9.0",
     "cpr": "~0.3.2",
     "deep-extend": "~0.2.8",
     "lodash": "~2.4.1",

